config/v1/types_infrastructure.go

```diff
@@ -614,6 +625,15 @@ type OpenStackPlatformStatus struct {
 	// datacenter DNS, a DNS service is hosted as a static pod to serve those hostnames
 	// to the nodes in the cluster.
 	NodeDNSIP string `json:"nodeDNSIP,omitempty"`
+
+	// loadBalancerType defines if a internal or external load balancer
+	// is to be used for the openshift api and ingress endpoints.
+	// When set to Internal the static pods in charge of API and Ingress traffic load-balancing
+	// defined in machine config operator will be deployed.
```

Comment:
Couple of grammar nits
```suggestion
	// loadBalancerType defines if an internal or external load balancer
	// is to be used for the openshift api and ingress endpoints.
	// When set to Internal the static pods in charge of API and Ingress traffic load-balancing
	// defined in the machine config operator will be deployed.
```